src/tlo/methods/alri.py

```diff
@@ -2650,6 +2654,16 @@ def _try_treatment(antibiotic_indicated: Tuple[str], oxygen_indicated: bool) ->
             oxygen_available = self._get_cons('Oxygen_Therapy')
             oxygen_provided = (oxygen_available and oxygen_indicated)
 
+            # todo: should equipment only be logged if consumables are available?
+            # If individual requires oxygen, log equipment
+            if oxygen_indicated:
```

Comment:
```suggestion
            if oxygen_provided:
```

@timcolbourn, I think the answer to the `# todo` above is yes, isn't? The equipment will be used only if `oxygen_provided` is `True`, am I right?

Reply:
updated as this for now.